src/ui/map.js

```diff
@@ -769,6 +778,21 @@ class Map extends Camera {
     }
 
     /**
+     * Returns a Boolean indicating whether a tile is loaded.
+     *
+     * @param {string} sourceId The ID of the source to be checked.
+     * @param {number} tileCoordId The ID of the source to be checked.
+     * @returns {boolean} A Boolean indicating whether the source is loaded.
+     */
+
+    isTileLoaded(sourceId, tileCoordId){
```

Comment:
yeah I was wondering about exposing how the TileCoord#id is computed... I'm fine w leaving this out too. I can't think of a user who would care about an individual tile as opposed to all tiles in viewport 💭